mlflow/tracing/types/wrapper.py

```diff
@@ -18,13 +17,14 @@ class MLflowSpanWrapper:
     Span object, so need to implement the same interfaces as the original Span.
     """
 
-    def __init__(self, span: trace_api.Span, span_type: SpanType = None):
+    def __init__(self, span: trace_api.Span, span_type: SpanType = SpanType.UNKNOWN):
```

Comment:
```suggestion
    def __init__(self, span: trace_api.Span, span_type: str = SpanType.UNKNOWN):
```

because we use `str` in other places?

Reply:
ah forgot to replace, thanks for the catch!